components/molecules/SonificationControls/PlayPauseButton/index.js

```diff
@@ -0,0 +1,29 @@
+"use client";
+import { useState } from "react";
+import IconButton from "@/components/atomic/IconButton";
+import styles from "./styles.module.css";
+import parameters from "@/components/organisms/Listener/parameters";
+
+const PlayPauseButton = () => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const handleClick = () => {
+    const newState = !isPlaying;
+    setIsPlaying(newState);
```

Comment:
handy React-ism for toggling a boolean state: `setIsPlaying(playing => !playing)`